Sources/Hub/HubApi.swift

```diff
@@ -8,21 +8,22 @@
 import Foundation
 
 public struct HubApi {
-    let endpoint = "https://huggingface.co/api"
     var downloadBase: URL
-    var hfToken: String? = nil
+    var hfToken: String?
+    var hfEndpoint: String
```

Comment:
Why not keep `endpoint` here?

Reply:
Because the token also has the "hf" prefix.😂